File: taskrouter/v1/workers_cumulative_statistics/fetch-default/fetch-default.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

workers_cumulative_statistics = @client.taskrouter
  .workspaces('WSaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa')
  .workers('WKaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa')
  .cumulative_statistics
  .fetch

puts workers_cumulative_statistics.account_sid

File: api/v2010/participant/fetch-default/fetch-default.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account.Conference;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var participant = ParticipantResource.Fetch(
            pathConferenceSid: "CFaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
            pathCallSid: "CAaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
        );

        Console.WriteLine(participant.AccountSid);
    }
}